.github/workflows/feature_6_docs.yml

```
@@ -0,0 +1,63 @@
+name: Develop > Tiny docs 6.0
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - '*'
+    branches:
+      - 'feature-6/**'
+
+jobs:
+  build:
+    name: Build Docs and Deploy
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [14]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        cache: 'yarn'
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Antora@3.0.1
+      run: sudo yarn global add @antora/cli@3.0.1
+
+    - name: (Feature - Docs 6.0) Install dependencies
+      run: yarn install -g
+
+    - name: (Feature - Docs 6.0) Build API References
+      run: yarn build
+
+    - name: (Feature - Docs 6.0) Build Website
+      run: antora ./antora-playbook-staging.yml
```

We don't need to use the antora global here, as we should be using the one installed by yarn. As such, I'd suggest doing something like this instead:
```suggestion
      run: yarn antora ./antora-playbook-staging.yml
```

---

Also should this be using the `dev` playbook instead of the `staging` one?

---

absolutely !